tests/integration/mininet_tests.py

```diff
@@ -155,17 +154,14 @@ def verify_events_log(self, event_log, timeout=10):
 
     def test_untagged(self):
         """All hosts on the same untagged VLAN should have connectivity."""
-        self._set_event_log(self.EVENT_LOGGER_TIMEOUT)
+        self._enable_event_log(10)
```

Comment:
why 10?

Reply:
Yeah modified that. I was too tired and made a mistake. One final review? @grafnu The tests pass now